src/parameters/define_parameter_test.ts

```diff
@@ -0,0 +1,40 @@
+import { DefineProperty } from "./define_parameter.ts";
```

Comment:
Need to rename the file we're importing from, and this test file

Reply:
Fixed in 9b73d87